docs/imaging/dlc.rst

```diff
@@ -0,0 +1,114 @@
+.. _Imaging dlc:
+
+=========================
+Imaging: DLC
+=========================
+
+`DeepLabCut <http://www.mackenziemathislab.org/deeplabcut>`_ is a software package for inferring motion capture information from tracking video. Whereas Optitrack makes use of one infra-red reflectors attached to the subject(s), DeepLabCut instead uses a deep neural network trained on human-labelled data of the subject(s), allowing for less interference with subject behaviour. 
+
+
+
+Schema
+------------
+
+The DLC implementation in the imaging pipeline is shown below
+
+.. figure:: /_static/imaging/dlc_schema.png
+   :alt: DLC in the Imaging schema
+   
+   DLC in the Imaging schema
+
+* ``DLCTrackingType``
+
+  Keeps track of the kinds of tracking to which DLC can be assigned. At present, DLC data is only supported for *openfield* tasks
+
+* ``DLCModel``
+
+  Contains the full specification of a trained DLC model used in prediction. Due to the way DLC is designed, this data is derived from several separate files.
+  When creating a new Method, please pay careful attention to how body parts are named (see ``TrackedBodyPart``)
+
+* ``DLCTrackingProcessingMethod``
+
+  DLC by itself keeps track of individual body parts, not an entire subject; while the imaging pipeline is built around the assumption that a subject can be assigned to a single state vector at any given time. This table keeps track of the method to calculate that state vector given the set of body part co-ordinates provided by DLC. Note that it has to know the *exact* name of each body part in question, and so if models use new names for the same body part, a new tracking processing method must be provided. 
+
+* ``SessionDLC``
+
+  Maps a specific Sessionto a specific Processing Method
+
+* ``TrackedBodyPart``
+
+  A list of the body parts that can be tracked. Please try to avoid adding to this list unnecessarily and follow the naming schems already in use
```

Comment:
... unnecessarily (e.g. "Left ear" vs. "LEFTEAR" - case and space sensitive namings!) ...